site/content/integrate/apps/deployment-aws/deploy-to-aws/_index.md

```diff
@@ -0,0 +1,131 @@
+
+#### 3. Set AWS Provision environment variables
+
+Open a terminal where you installed the Apps plugin and set the following variables to the AWS credentials just created and saved:
+
+- `MM_APPS_PROVISION_AWS_ACCESS_KEY`  
+- `MM_APPS_PROVISION_AWS_SECRET_KEY`
+
+Set the following environment variables based on bucket name and region
+
+- `MM_APPS_S3_BUCKET`  
+- `MM_APPS_AWS_REGION`
+
+#### 4. Initialize the AWS resources
+
+The following command will create a Mattermost invoke user and security policy for use with AWS.
+
+`go run ./cmd/appsctl aws init --create --create-access-key`
+
+The output of the command will contain two "Invoke" environment variables.
+
+#### 5. Set AWS Invoke environment variables
+
+The second set of credentials created from the above initialization step will be used by Mattermost to invoke Apps.  Set these variables in the location of your running Mattermost server repo.
+
+- `MM_APPS_AWS_ACCESS_KEY`
+- `MM_APPS_AWS_SECRET_KEY`
+
+### Restart the Mattermost Server
+
+Ensure you have the following environment variables set.
+
+- `MM_APPS_AWS_ACCESS_KEY`
+- `MM_APPS_AWS_SECRET_KEY`
+- `MM_APPS_S3_BUCKET`
+- `MM_APPS_AWS_REGION`
+
+Restart the Mattermost server to complete your Mattermost and AWS setup. You can now deploy an app to AWS.
+
+### Test your AWS configuration and permissions
+
+Run the following commands to test and validate your AWS configuration and permissions.
+
+```bash
+go run ./cmd/appsctl aws test lambda
+go run ./cmd/appsctl aws test s3
+```
+
+## Deploying an app to AWS
+
+After setting up your Mattermost instance, AWS key, and S3 bucket you can now provision your app using `appsctl`. Note that `appsctl` commands are run in the `mattermost-plugin-apps` repo.
+
+### Install appsctl  
+
+`go install github.com/mattermost/mattermost-plugin-apps/cmd/appsctl@latest`
+
+### Provision your app  
+
+`appsctl provision app /PATH/TO/YOUR/APP/BUNDLE.zip`
+
+### Install your app  
+
+The provision command output will show the Mattermost `/apps install` installation command. You can also install a previously provisioned app with the same `/apps install` command. (Example: `/apps install aws hello-lambda demo`)
+
+# Provisioned app details
+
+AWS Lambda functions have semantic names, which means that a function described in the `manifest.json` file translates to AWS as `$appID_$appVersion_$functionName` to avoid collisions with other apps' or other versions' functions. And **appsctl** provisions lambda functions using this name. For example the name of a `servicenow` app's lambda function might be `com-mattermost-servicenow_0-1-0_go-function`. You don't need to worry about the AWS Lambda function names, as **appsctl** takes care of it. The dedicated S3 bucket name is stored in the environment variable: `MM_APPS_S3_BUCKET`.
```

Comment:
nit: not sure how formal we should be in general, but I'd avoid expressions like "takes care of it" and replace them with smth. like "handles it".

Reply:
Fixed in ce6262cf